### ISSUE TITLE
Compilation error of PyBulletPlugin

### ISSUE THREAD

**Issue by bchretien** [s-nakaoka/choreonoid#19]
While trying to compile the `master` branch of choreonoid on Arch Linux, I encountered the following build error:

```sh
Scanning dependencies of target PyPythonSimScriptPlugin
[ 98%] Building CXX object src/PythonSimScriptPlugin/python/CMakeFiles/PyPythonSimScriptPlugin.dir/PyPythonSimScriptPlugin.cpp.o
/tmp/makepkg/choreonoid-git/src/choreonoid/src/BulletPlugin/python/PyBulletPlugin.cpp: In function ‘void cnoid::init_module_BulletPlugin()’:
/tmp/makepkg/choreonoid-git/src/choreonoid/src/BulletPlugin/python/PyBulletPlugin.cpp:40:63: error: ‘createInstance’ was not declared in this scope
             .def("__init__", boost::python::make_constructor(&createInstance<BulletSimulatorItem>));
                                                               ^
/tmp/makepkg/choreonoid-git/src/choreonoid/src/BulletPlugin/python/PyBulletPlugin.cpp:40:97: error: expected primary-expression before ‘>’ token
             .def("__init__", boost::python::make_constructor(&createInstance<BulletSimulatorItem>));
                                                                                                 ^
/tmp/makepkg/choreonoid-git/src/choreonoid/src/BulletPlugin/python/PyBulletPlugin.cpp:40:98: error: expected primary-expression before ‘)’ token
             .def("__init__", boost::python::make_constructor(&createInstance<BulletSimulatorItem>));
                                                                                                  ^
src/BulletPlugin/python/CMakeFiles/PyBulletPlugin.dir/build.make:54: recipe for target 'src/BulletPlugin/python/CMakeFiles/PyBulletPlugin.dir/PyBulletPlugin.cpp.o' failed
make[2]: *** [src/BulletPlugin/python/CMakeFiles/PyBulletPlugin.dir/PyBulletPlugin.cpp.o] Error 1
CMakeFiles/Makefile2:1341: recipe for target 'src/BulletPlugin/python/CMakeFiles/PyBulletPlugin.dir/all' failed
make[1]: *** [src/BulletPlugin/python/CMakeFiles/PyBulletPlugin.dir/all] Error 2
```

Boost 1.58, Bullet 2.82, Python 2.7.9.

**Comment by bchretien** [comment]
I guess this was missed when removing the method in 027e30baa333134ed622567a8ec31813d457148e.

**Comment by bchretien** [comment]
Solved in #24.
